tests/test_psf.py

```diff
@@ -219,6 +220,8 @@ def setupDeterminer(
             psfDeterminerConfig.stampSize = stampSize
         psfDeterminerConfig.debugStarData = debugStarData
         psfDeterminerConfig.useCoordinates = useCoordinates
+        if downsample:
+            psfDeterminerConfig.maxCandidates = 10
```

Comment:
I'd have imagined passing `maxCandidates` as the optional argument instead of `downsample`. Is this choice because `chiLim` value below depends on `maxCandidates`?